frappe/boot.py

```diff
@@ -68,6 +68,9 @@ def get_bootinfo():
 	bootinfo.home_folder = frappe.db.get_value("File", {"is_home_folder": 1})
 	bootinfo.navbar_settings = get_navbar_settings()
 	bootinfo.notification_settings = get_notification_settings()
+	bootinfo.onboarding_tours = frappe.parse_json(
+		frappe.db.get_single_value("Form Tour Settings", "onboarding_tours") or "[]"
+	)
```

Comment:
We should not load this every time. If logged-in users have no remaining onboarding steps, we should return []. We should keep a flag all_onboarding_completed for each user.

Reply:
We would have to call twice one to check if all are completed and other to get tours if not. also making sure that all_tours_completed is updated on various occasions. right now it doesn't seem that big of a issue we can implement something if we have many more tours and face any issues?